src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/UserService/IUserService.cs

```diff
@@ -7,5 +7,9 @@ namespace SFA.DAS.LevyTransferMatching.Infrastructure.Services.UserService
     public interface IUserService
     {
         Task<IEnumerable<UserAccountDto>> GetLoggedInUserAccounts();
+
+        string GetUserId();
+
+        string GetUserDisplayName();
```

Comment:
In the APIM review, I mentioned we could probably put all of this kind of stuff into one model that gets passed around - given the two _are_ related, you could combine these two methods into one.